conversions/src/Data/Conversions.hs

```diff
@@ -136,6 +140,21 @@ instance Conversion Int Word8 where
 instance Conversion Int Word16 where
   convert = fromIntegral
 
+instance Conversion Int Word32 where
+  convert = fromIntegral
+
+-- do something better
+instance (MonadError (BoundError Word64 Int) m) => Conversion (m Int) Word64 where
+  convert value = do
+    when (value > convertImpure (maxBound @Int)) -- TODO: standardize into helper and re-use
```

Comment:
instance of https://github.com/mbj/mhs/pull/243/commits/900e4ebf304c13d2143ce6721f8b2ab787002618#r1231396417